org.adempiere.base/src/org/adempiere/model/MWlistboxCustomization.java

```diff
@@ -0,0 +1,152 @@
+ * or via info@compiere.org or http://www.compiere.org/license.html           *
+ *****************************************************************************/
+package org.adempiere.model;
+
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Properties;
+
+import org.compiere.model.Query;
+import org.compiere.model.X_AD_Wlistbox_Customization;
+
+public class MWlistboxCustomization extends X_AD_Wlistbox_Customization {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -3220641197739038436L;
+
+	/**
+	 * @param ctx
+	 * @param AD_WListboxName
+	 * @param AD_Wlistbox_Customization_ID
+	 * @param trxName
+	 * @return
+	 */
+	public MWlistboxCustomization(Properties ctx, int AD_Wlistbox_Customization_ID, String trxName) {
+		super(ctx, AD_Wlistbox_Customization_ID, trxName);
+	}
+
+	/**
+	 * @param ctx
+	 * @param rs
+	 * @param trxName
+	 * @return
+	 */	
+	public MWlistboxCustomization(Properties ctx, ResultSet rs, String trxName) {
+		super(ctx, rs, trxName);
+	}
+
+	/**
+	 * @param ctx
+	 * @param AD_User_ID
+	 * @param AD_WListboxName
+	 * @param trxName
+	 * @return
+	 */	
+	public static MWlistboxCustomization get(Properties 	ctx, 
+											 int 			AD_User_ID, 
+											 String 		AD_WListboxName, 
+											 String 		trxName)
+	{
+		Query query = new Query(ctx, Table_Name, "AD_User_ID=? AND WListboxName=?", trxName);
+		return query.setClient_ID().setOnlyActiveRecords(true).setParameters(new Object[] { AD_User_ID, AD_WListboxName}).first();
+	}
+
+
+	private static ArrayList<String> cleanCustomization (String[] CustomizationNew)
+	{
+		for (int i = 0; i < CustomizationNew.length; i++ )
+		{
+			CustomizationNew[i] = CustomizationNew[i].substring(0, CustomizationNew[i].indexOf("=") );
+		}
+		return new ArrayList<String>(Arrays.asList(CustomizationNew)); 
+	}	
+
+	private static int columnIndex(String 				searchColumnName, 
+								   ArrayList<String> 	columnList)
+	{
+		for (int i = 0; i < columnList.size(); i++)
+		{
+			if (columnList.get(i).equals(searchColumnName))
+			{
+				return i;
+			}
+		}
+	return -1;
+	}
+
+	/**
+	 * Save the columnWidth of the columns of the WListBox 
+	 * @param ctx
+	 * @param AD_WListboxName
+	 * @param AD_User_ID
+	 * @param Custom
+	 * @param trxName
+	 * @return
+	 */
+	
+	public static void saveData(Properties  ctx,
+								String      AD_WListboxName, 
+								int         AD_User_ID, 
+								String      Custom, 
+								String      trxName) 
+	{	
+		MWlistboxCustomization WlistBoxCust = get(ctx, AD_User_ID, AD_WListboxName, trxName);
+	
+		if (WlistBoxCust != null && WlistBoxCust.getAD_Wlistbox_Customization_ID()> 0)
+		{
+			String[] custOrg = WlistBoxCust.getCustom().split(",");
+			ArrayList<String> newColumnList = new ArrayList<>( Arrays.asList(Custom.trim().split(",")) );
+			ArrayList<String> orgColumnList  = new ArrayList<>( Arrays.asList(WlistBoxCust.getCustom().trim().split(",")) );
```

Comment:
the use of `new ArrayList<>(...)` here is unnecessary, use List<String> instead.

Reply:
Done

Reply:
hi @gsjeffen , why was this being changed back to the "new ArrayList" approach ?

Reply:
Hi @hensin

It is only orgColumnList that is changed

List<String> newColumnList =  Arrays.asList(Custom.trim().split(","));
ArrayList<String> orgColumnList  = new ArrayList<>( Arrays.asList(WlistBoxCust.getCustom().trim().split(",")) );

This is due to the following:
orgColumnList.addAll(addColumn);

if orgColumnList is of list then this would give an exception.